### PR TITLE
[D&D] Initial functional tests

### PR DESCRIPTION
**Pull request by ashwin-pc**
### Description
- Adds a few basic functional tests and sets up the page objects for running functional tests for the Wizard editor
- Updates readme for testing with details for chrome version
- Fixes a bug with the datasource selector styling
 
### Check List
- [x] New functionality includes testing.
  - [x] All tests pass
    - [ ] `yarn test:jest`
    - [ ] `yarn test:jest_integration`
    - [ ] `yarn test:ftr`
- [ ] New functionality has been documented.
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: test/functional/page_objects/wizard_page.ts [hunk 1]
```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { FtrProviderContext } from '../ftr_provider_context';
+// import { VisualizeConstants } from '../../../src/plugins/visualize/public/application/visualize_constants';
```
Comment (ananzh) [inline test/functional/page_objects/wizard_page.ts:7]: NIT: should we remove this line?

File: TESTING.md [hunk 2]
```diff
@@ -50,7 +50,7 @@ To run specific functional tests, you can run by CI group:
 
 To debug functional tests:
 Say that you would want to debug a test in CI group 1, you can run the following command in your environment:
-`node --debug-brk --inspect scripts/functional_tests.js --config test/functional/config.js --include ciGroup1 --debug`
```
Comment (kavilla) [inline TESTING.md:53]: Does this work!? I know it starts for me but it doesn't really help.

File: TESTING.md [hunk 3]
```diff
@@ -89,9 +89,24 @@ Automated testing is provided with Jenkins for Continuous Integration. Jenkins e
 Selenium tests are run in headless mode on CI. Locally the same tests will be executed in a real browser. You can activate headless mode by setting the environment variable:
 `export TEST_BROWSER_HEADLESS=1`
 
+Since local Selenium tests are run in a real browser, the dev environment should have a desktop environment and a Google Chrome installed to run the tests.
```
Comment (kavilla) [inline TESTING.md:92]: I believe Google Chrome isn't explicit? Like I believe Chromium is even supported.

File: TESTING.md [hunk 4]
```diff
@@ -89,9 +89,24 @@ Automated testing is provided with Jenkins for Continuous Integration. Jenkins e
...
 By default the version of OpenSearch Dashboards will pull the snapshot of the same version of OpenSearch if available while running a few integration tests and for running functional tests. However, if the version of OpenSearch Dashboards is not available, you can build OpenSearch locally and point the functional test runner to the executable with:
 `export TEST_OPENSEARCH_FROM=[local directory of OpenSearch executable]`
 
+Selinium tests require a chromedriver and a corresponding version of chrome to run properly. Depending on the version of chromedriver used, you may need to use a version of Google Chrome that is not the latest version. You can do this by running:
+
+```sh
+# Enter tghe version of chrome that you want to install
+CHROME_VERSION=100.0.4896.127-1
+
+# Download Chrome to a temp directory
```
Comment (kavilla) [inline TESTING.md:103]: NIT: specific to amd64 right maybe we can note it.

File: src/plugins/wizard/public/application/components/data_tab/dropbox.tsx [hunk 5]
```diff
@@ -126,6 +133,7 @@ const DropboxComponent = ({
                 aria-label="clear-field"
                 iconSize="s"
                 onClick={() => onAddField()}
+                data-test-subj="dropBoxAddBtn"
```
Comment (kavilla) [inline src/plugins/wizard/public/application/components/data_tab/dropbox.tsx:136]: NIT: i see current standard has `camelButton` or `kebab-button`

File: test/functional/apps/wizard/_experimental_vis.ts [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+      });
+      await PageObjects.visualize.navigateToNewVisualization();
+      await PageObjects.visualize.waitForVisualizationSelectPage();
+
+      // Try to find the wizard Vis type.
+      const wizardVisTypeExists = await PageObjects.visualize.hasVisType('wizard');
+      expect(wizardVisTypeExists).to.be(false);
+
+      // unset the experimental ui setting
+      await opensearchDashboardsServer.uiSettings.unset(VISUALIZE_ENABLE_LABS_SETTING);
```
Comment (kavilla) [inline test/functional/apps/wizard/_experimental_vis.ts:46]: should we include an after step incase this test fails?

**Comment by kavilla** [comment]
I believe some folks still use the [Jenkinsfile](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/Jenkinsfile#L70)

**Review by kavilla — COMMENTED** [review]
just some NITs, and protective testing but looks goood! the test subject I think is the most important part and we can keep adding more in the ftuure.

File: TESTING.md (hunk 2 above)
Comment (ashwin-pc) [reply TESTING.md:53]: yep. thats why I changed the flag name

File: TESTING.md (hunk 3 above)
Comment (ashwin-pc) [reply TESTING.md:92]: Okay, ill update that 👍

File: src/plugins/wizard/public/application/components/data_tab/dropbox.tsx (hunk 5 above)
Comment (ashwin-pc) [reply src/plugins/wizard/public/application/components/data_tab/dropbox.tsx:136]: Our styleguide prescribes `cameCase` with `-` for suffixing the subject with an id of sorts. I've followed that pattern here.

File: TESTING.md (hunk 4 above)
Comment (ashwin-pc) [reply TESTING.md:103]: Are you sure? `amd64` software can be installed on both intel and AMD cpu's as long as they support the same instruction set, which all the processors released in the last decade do

File: test/functional/apps/wizard/_experimental_vis.ts (hunk 6 above)
Comment (ashwin-pc) [reply test/functional/apps/wizard/_experimental_vis.ts:48]: done

**Review by kavilla — APPROVED** [review]

**Pull request merged**